packages/web-client/app/services/card-customization.ts

```diff
@@ -81,8 +81,15 @@ export default class CardCustomization extends Service {
 
   async ensureCustomizationOptionsLoaded() {
     if (!this.loaded) {
-      return taskFor(this.fetchCustomizationOptionsTask).perform();
+      try {
+        await taskFor(this.fetchCustomizationOptionsTask).perform();
+      } catch (e) {
+        if (!didCancel(e)) {
+          throw e;
+        }
+      }
     }
+    return Promise.resolve();
```

Comment:
We can omit this since `async` functions will return `Promise<void>` if there is no return statement.